src/components/common/header/PageHeader.tsx

```diff
@@ -69,7 +66,7 @@ function PageHeader({
                 )}
             </h1>
             {showTabs && renderHeaderTabs()}
-            {showCreateButton && renderCreateButton()}
+            {renderCreateButton && renderCreateButton()}
```

Comment:
Can we add functionType check here rather than renderCreateButton this?

Reply:
We should also rename this from renderCreateButton to renderCTAs or renderActionButtons may be. or something similar